### PR TITLE
Update view on refund details

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-811

This code reflect the latest changes made to the wireframe to inform the user about the correct amount that is going to be refunded
Screenshot 2020-01-09 at 14 45 34

### CODE REVIEW

File: app/presenters/refund_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    super(finance_details, nil)
+  end
+
+  def total_amount_paid
+    @_total_amount_paid ||= payments.inject(0) do |memo, payment|
+      memo + payment.amount
+    end
+  end
+
+  def amount_due
```
Comment (Cruikshanks) [inline app/presenters/refund_presenter.rb]: Could we amend the name to be consistent with the other method?

```suggestion
  def total_amount_due
```

File: app/presenters/refund_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    @_amount_due ||= orders.inject(0) do |memo, order|
```
Comment (cintamani) [inline app/presenters/refund_presenter.rb]:
```suggestion
    @_total_amount_due ||= orders.inject(0) do |memo, order|
```

File: app/views/refunds/new.html.erb [hunk 3]
```diff
@@ -16,17 +16,55 @@
...
+            <td class="numeric">
+              £<%= display_pence_as_pounds_and_cents(@presenter.total_amount_paid) %>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <%= t(".total_amount_due") %>
+            </td>
+            <td class="numeric">
+              £<%= display_pence_as_pounds_and_cents(@presenter.amount_due) %>
```
Comment (cintamani) [inline app/views/refunds/new.html.erb]:
```suggestion
              £<%= display_pence_as_pounds_and_cents(@presenter.total_amount_due) %>
```

File: spec/presenters/refund_presenter_spec.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    let(:payment1) { double(:payment, amount: 100) }
+    let(:payment2) { double(:payment, amount: 600) }
+    let(:finance_details) { double(:finance_details, payments: [payment1, payment2]) }
+
+    it "returns the total of all payments made by a user to a registration" do
+      expect(subject.total_amount_paid).to eq(700)
+    end
+  end
+
+  describe ".amount_due" do
```
Comment (cintamani) [inline spec/presenters/refund_presenter_spec.rb]:
```suggestion
  describe ".total_amount_due" do
```

File: spec/presenters/refund_presenter_spec.rb [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    let(:order1) { double(:order, total_amount: 100) }
+    let(:order2) { double(:order, total_amount: 600) }
+    let(:finance_details) { double(:finance_details, orders: [order1, order2]) }
+
+    it "returns the total amount of all orders made by a user to a registration" do
+      expect(subject.amount_due).to eq(700)
```
Comment (cintamani) [inline spec/presenters/refund_presenter_spec.rb]:
```suggestion
      expect(subject.total_amount_due).to eq(700)
```

**Pull request merged**